setup.py

```diff
@@ -39,6 +39,9 @@
             'azure==1.0.3'],
         'encryption': [
             'pynacl==0.3.0'],
+        'google': [
+            'gcs_oauth2_boto_plugin==1.9',
+            'boto==2.38.0'],
```

Comment:
Can you factor this out into a variable to ensure that the `aws` and `google` extras don't use conflicting versions?